src/shared/services/roots.js

```diff
@@ -7,17 +7,29 @@ import { getAuthToken } from 'shared/utils/storage';
 
 import axios from 'axios';
 
-const httpConfig = {
-  baseURL: getBaseURL(),
-  withCredentials: true, // Indicates whether or not cross-site Access-Control requests should be made using credentials
-  xsrfHeaderName: HEADER_CSRF_TOKEN, // the name of the http header that carries the xsrf token value
-  xsrfCookieName: 'csrftoken', // The name of the cookie to use as a value for xsrf token
-  timeout: 30000, // If the request takes longer than `timeout`, the request will be aborted (Error: timeout of 1000ms exceeded)
-  transformResponse: [data => ({ ...JSON.parse(data) /* , timeStamp: new Date() */ })], // Changes to the response to be made before it is passed to then/catch
-  headers: { [HEADER_SOURCE]: 'mobile' }, // Custom headers to be sent
+// const httpConfig = {
+//   baseURL: getBaseURL(),
+//   withCredentials: true, // Indicates whether or not cross-site Access-Control requests should be made using credentials
+//   xsrfHeaderName: HEADER_CSRF_TOKEN, // the name of the http header that carries the xsrf token value
+//   xsrfCookieName: 'csrftoken', // The name of the cookie to use as a value for xsrf token
+//   timeout: 30000, // If the request takes longer than `timeout`, the request will be aborted (Error: timeout of 1000ms exceeded)
+//   transformResponse: [data => ({ ...JSON.parse(data) /* , timeStamp: new Date() */ })], // Changes to the response to be made before it is passed to then/catch
+//   headers: { [HEADER_SOURCE]: 'mobile' }, // Custom headers to be sent
+// };
+
+// const fetch = axios.create(httpConfig);
+
+const httpAnalyticsServerConfig = {
+    baseURL: getBaseURL(),
+    withCredentials: true, // Indicates whether or not cross-site Access-Control requests should be made using credentials
+    xsrfHeaderName: HEADER_CSRF_TOKEN, // the name of the http header that carries the xsrf token value
+    xsrfCookieName: 'csrftoken', // The name of the cookie to use as a value for xsrf token
+    timeout: 30000, // If the request takes longer than `timeout`, the request will be aborted (Error: timeout of 1000ms exceeded)
+    transformResponse: [data => ({ ...JSON.parse(data) /* , timeStamp: new Date() */ })], // Changes to the response to be made before it is passed to then/catch
+    headers: { [HEADER_SOURCE]: 'mobile' }, // Custom headers to be sent
 };
 
-const fetch = axios.create(httpConfig);
+const fetch = axios.create(httpAnalyticsServerConfig);
```

Comment:
You should use a separate `fetch` object. Doing so will affect all the API requests in this project.